.github/workflows/ci.yaml

```diff
@@ -321,3 +321,20 @@ jobs:
       uses: ./java-shared-dependencies/unmanaged-dependency-check
       with:
         bom-path: gapic-generator-java-bom/pom.xml
+
+  renovate_config_check:
```

Comment:
Can you make this as independent workflow?

Then I think of 2 options:

- workflow dispatch to periodically validate the config files in ~13 repositories. It's installed in sdk-platform-java.
- workflow for pull requests that touches renovate.json. It's installed in every ~13 repositories via synthtool template.

What is the trade off?

Reply:
synthool is better, periodically validating makes no sense if we can run it upon every touch to renovate.json
Raised https://github.com/googleapis/synthtool/pull/1920

Reply:
Trade off: 
1) Using manual workflow dispatch for periodic validation is not a good idea because 
   (a) Manual invocation required everytime 
   (b) If there is an error in the config, we would get to know about it only when we run this check manually.
2) Using synthtool, this check will be installed as an independent workflow in each repository, running on every commit to config file, hence catching errors as and when they are about to be committed. This way we can catch the error faster and better. 

Hence going with synthool.